src/contentmanager.cpp

```diff
@@ -132,6 +137,12 @@ void ContentManager::onCustomContextMenu(const QPoint &point)
     connect(&menuResumeBook, &QAction::triggered, [=]() {
         resumeBook(id, index);
     });
+    connect(&menuOpenFolder, &QAction::triggered, [=]() {
+        if (bookPath != "") {
+            QFileInfo fileInfo(bookPath);
+            QDesktopServices::openUrl(fileInfo.absoluteDir().absolutePath());
+        }
+    });
```

Comment:
It is better to setup the action in the "try/catch" instead of creating a "global" `bookPath` and setting if `if` in the action.